automation/vars/uni-delta.yaml

```diff
@@ -0,0 +1,55 @@
+---
+vas:
+  uni-delta:
+    stages:
+      - path: examples/dt/uni-delta/control-plane/nncp
+        validations:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni-delta/control-plane
+        validations:
+          - >-
+            oc -n openstack wait osctlplane controlplane
+            --for condition=Ready
+            --timeout=600s
+        values:
+          - name: network-values
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni-delta/ceph
+        validations:
+          - >-
+            oc -n openstack wait osdpd
+            edpm-ceph
+            --for condition=Ready
+            --timeout=1200s
+        values:
+          - name: edpm-ceph-values
+            src: values.yaml
+        build_output: edpm-ceph.yaml
+        post_stage_run:
+          - name: Deploy Ceph
+            type: playbook
+            source: "../../playbooks/ceph.yml"
+            inventory: "${HOME}/ci-framework-data/artifacts/zuul_inventory.yml"
```

Comment:
This would be the ceph group inventory file `"${HOME}/reproducer-inventory/ceph-group.yml"`